crcls/blocksui-marketplace
# feat: add block controls

## Purpose

1. Added up and down controls to blocks, so blocks can be moved. These are only present on blocks that accept other blocks right now. I will need to go back and update the styling of the other blocks, so they can have these and future controls such as a delete and options.
2. Added the start of color highlighting when dropping a block.

## Testing Instructions

1. Confirm the up control moves a button up.
2. Confirm the down control moves a button down.
3. Confirm the main drop zone highlights light green when a block is dragged over it.
4. Confirm blocks that accept other blocks highlight light green when a block is dragged over it.

## Checklist
